### PR TITLE
EDU-1066: Clarify that pricing includes support

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

File: docs-src/cloud/pricing-support.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ tags:
   - explanation
 ---
 
-With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
+When signing up for Temporal Cloud, you select a support plan—either Basic or Premium—and gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
```
Comment (rachfop) [inline docs-src/cloud/pricing-support.md:13]:
```suggestion
When signing up for Temporal Cloud, select a support plan—either Basic or Premium—and gain access to our support organization, which includes developer success engineers and solution architects. Our experts assist with a range of work streams, from Workflow design reviews to setting up observability, and from break/fix support to maintaining an agreed-upon set of SLAs.
```

Comment (rachfop) [reply docs-src/cloud/pricing-support.md:13]: I know you didn't write it, but I really don't like the `break/fix` line. Would troubleshooting be better?
We aren't dividing break by fix.

File: docs-src/cloud/pricing-support.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ tags:
   - explanation
 ---
 
-With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
+When signing up for Temporal Cloud, select a support plan—either Basic or Premium—and gain access to our support organization, which includes developer success engineers and solution architects. Our experts assist with a range of work streams, from Workflow design reviews to setting up observability, and from break/fix support to maintaining an agreed-upon set of SLAs.
```
Comment (lorensr) [inline docs-src/cloud/pricing-support.md:13]:
```suggestion
When signing up for Temporal Cloud, select a support plan—either Basic or Premium—and gain access to our support organization, which includes developer success engineers and solution architects. Our experts assist with a range of work streams, from Workflow design reviews to setting up observability, and from troubleshooting support to maintaining an agreed-upon set of SLAs.
```

Comment (rachfop) [reply docs-src/cloud/pricing-support.md:13]: Like it.

**Review by rachfop — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Comment by flossypurse** [comment]
Ah, this appears to be a duplicate effort already resolved here: https://github.com/temporalio/documentation/pull/2358/

**Comment by lorensr** [comment]
@flossypurse I've read that PR, and believe merging this PR would still be helpful in clarifying. Users are still asking about it.

**Pull request merged**